cloud-inquisitor/notification/test_text/hijack_test_with_chain.txt

```diff
@@ -0,0 +1,24 @@
+Potential Domain Hijack Found
+
+Deleting resource S3 Bucket:test.bucket.com in AWS account 12345678 has resulted in a potential domain hijack;
+ there is a DNS record pointing to a resource not owned or no longer owned by us.
+
+The following table includes all resources in the chain that would lead to a potential domain hijack.
+
+
+
+Resource 0:
+    Account ID:                 abcdefg
+    Resource:                   public.test.bucket.com
+    Resource Type:              route53
+    Resource Referenced:        test.bucket.com
+    Resource Referenced Type:   S3 Bucket
+
+Resource 1:
+    Account ID:                 abcdefg
+    Resource:                   public.test.bucket.com
+    Resource Type:              route53
+    Resource Referenced:        test.bucket.com
+    Resource Referenced Type:   S3 Bucket
+
```

Comment:
Also if we can include the friendly account name as well that would be desired. Some people are gatekeeping a number of accounts so this eliminates any confusion

Reply:
We dont currently have a way to map a user friendly name to an AWS account id. I can create an issue for future support; or we can re-scope and include this.

Reply:
Fair, in some environments, the account-name per profile/billing isn't always configured so this is a better value.